FirebaseFirestoreSwift.podspec

```diff
@@ -37,4 +37,9 @@ Google Cloud Firestore is a NoSQL document database built for automatic scaling,
   ]
 
   s.dependency 'FirebaseFirestore', '~> 1.6', '>= 1.6.1'
+
+  s.test_spec 'unit' do |unit_tests|
+    # The other Swift tests require FSTDocRef, gtest, and other dependencies.
+    unit_tests.source_files = 'Firestore/Swift/Tests/API/*.swift'
```

Comment:
I'm not enthusiastic about this. It suggests that this is a valid way to test this pod, but it's really not. It's skipping all the hard parts that do most of the heavy lifting.

The way to test this code is via the Firestore/Example/Podfile, which fully exercises all of it.

Reply:
Yep, I was hoping that it would be feasible to include more of the tests here.

The counter argument is that the test spec will lower the bar to contribute Swift-only extensions.  @ryanwilson Any insights about this after investigating Firestore and combine?

Reply:
There are no Swift-only things you can write that don't in some way depend upon the core Firestore API. The only way to write those kinds of tests today is to use the test helpers out of the Firestore testutils.